source-code/sdk-js/src/runtime/runtime.test.ts

```diff
@@ -1,9 +1,45 @@
+import type { Message, Resource } from "@inlang/core/ast"
 import { describe, expect, test } from "vitest"
-import { initBaseRuntime, initRuntime, RuntimeState } from "./runtime.js"
+import { initBaseRuntime, initRuntime, RuntimeContext, RuntimeState } from "./runtime.js"
+
+// this is a copy from `source-code/core/src/lint/linter.test.ts`
+// TODO: expose utility functions somewhere
```

Comment:
I already have exposed those functions somehwere, but @samuelstroschein reverted those changes. I would love to have them defined somewhere so we can use them to write tests faster.

Reply:
Yes, coming in `@inlang/core/tests`. That is part of https://github.com/inlang/inlang/pull/472.

Reply:
When is this ready?

Reply:
End of this week. Just copy & paste the function for now